Client/Frontend/Browser/TabScrollController.swift

```diff
@@ -252,9 +263,18 @@ extension TabScrollingController: UIScrollViewDelegate {
 
         suppressToolbarHiding = false
     }
-
+    
+    func scrollViewDidEndDecelerating(scrollView: UIScrollView) {
+        showOrHideWebViewContainerToolbar()
+    }
+    
+    func scrollViewDidEndZooming(scrollView: UIScrollView, withView view: UIView?, atScale scale: CGFloat) {
+        showOrHideWebViewContainerToolbar()
+    }
+    
```

Comment:
**Problem:**
If we zoom slightly by double-tapping on a non-text area. Then, scroll down just enough to hide the webviewcontainertoolbar toolbar and a little further. Now double-tap on the non-text area again. A blank block appears in place of the toolbar. 

As an example, go to this url: https://services.github.com/kit/downloads/github-git-cheat-sheet.pdf

1. While at the top of the page, double-click on the dark blue area (header of the sheet). It zooms slighly.
2. Now, scroll down until the WebViewController Toolbar just about disappears and slightly further.
3. Double click on the dark blue area again. The behavior that I see is: The content offset gets set to zero (I mean, the page goes back to its root position). And I see a blank block in place of the toolbar.